.github/workflows/format.yml

```
@@ -0,0 +1,33 @@
+
+      - name: Format code
+        run: |
+          cd expo-app
+          npx prettier . --write
+      - name: Commit and push changes
+        run: |
+          git config user.name "github-actions[bot]"
+          git config user.email "github-actions[bot]@users.noreply.github.com"
+          git add .
+          if [ -z "$(git status --porcelain)" ]; then
+            echo "No changes to commit"   
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)